Documentation/Security/GuidelinesAdministrators/ContentSecurityPolicy.rst

```diff
@@ -31,6 +31,11 @@ impractical for some sites) or add content security policy headers for
 these directories - basically all public available base directories of
 file storages (`sys_file_storage`).
 
+Please note that the CSP configuration mentioned in :ref:`content-security-policy`
+only applies to pages served by the TYPO3 Middleware. Direct access to
+files not served by TYPO3 (=PHP), as is the case with files in `fileadmin`, needs
+manual server configuration.
```

Comment:
Maybe explain why this should be necessary. I can think of svg to disable scripts.